### DISCUSSION TITLE
[Translation] - Monika After Story 0.12.2 translation into Português (Brazilian)

### DISCUSSION THREAD

**Discussion by RaitoninguHikari** [Monika-After-Story/MonikaModDev#7830] [general - translations]
I noticed that the site that hosted the translation of Monika After Story translated into Brazilian Portuguese, is down or no longer exists.
So, the purpose of the post is to update this with the currently functional site that keeps constant updates.
Translation team list: DDLC Translate Company Plus.
[Site](https://doki-doki-transtale-company.vercel.app/Mods/Monika%20After%20Story.html)

**Comment by RaitoninguHikari** [comment]
@ThePotatoGuy

**Comment by Moonlord458** [comment]
Can anyone tell me if the PT-BR translation will no longer continue?
